src/suricata.c

```diff
@@ -30,6 +30,10 @@
 #if HAVE_SIGNAL_H
 #include <signal.h>
 #endif
+#ifndef OS_WIN32
+// setrlimit
+#include <sys/resource.h>
```

Comment:
we have guards for header files, and we check them in configure. This one should be added as well.

Reply:
Ok

By the way
`git grep resource.h src/ | cut -d: -f1 | uniq | wc -l` gives 5
when
`git grep HAVE_SYS_RESOURCE_H src/ | cut -d: -f1 | uniq | wc -l` gives only 2

Reply:
Might be useful to do a similar check for other system headers.